app/views/projects/index.html.erb

```diff
@@ -1,9 +1,10 @@
-<h1>Projects</h1>
 <% @projects.each do |project| %>
   <h2><%= project.name %></h2>
     <% project.team_members.each do |team_member| %>
       <ul>
+        <%= image_tag team_member.thumbnail %>
         <li><%= team_member.name %></li>
+        <li><%= team_member.role %></li>
```

Comment:
What happens if either of these are missing from the fetched data? There are no validations on the data, so it's currently allowed for `role` and `thumbnail` to be `nil`.

Reply:
At the moment the return of nil is fine. It just means it doesn't display anything on the views page.
We will ensure the validations are added in a later branch once we have the basic first pass for the dashboard

Reply:
Fine :)